src/components/ContentSidebar/ActivityFeed/comment/Comment.js

```diff
@@ -115,9 +114,8 @@ class Comment extends React.Component<Props, State> {
         const { toEdit } = this;
         const { isEditing, isFocused, isInputOpen } = this.state;
         const createdAtTimestamp = new Date(created_at).getTime();
-        const canDelete = getProp(permissions, 'can_delete', false);
-        const canEdit = getProp(permissions, 'can_edit', false);
         const createdByUser = created_by || PLACEHOLDER_USER;
+        const isCommentAuthor = currentUser && createdByUser.id === currentUser.id;
```

Comment:
side note from working on permissions at progressly - it helps to keep the permissions questions separate from other logic, so even if this computation isn't isolated at least the result can be phrased as a "canDoX" variable name